packages/react/src/components/Masthead/MastheadLeftNav.js

```diff
@@ -48,7 +54,8 @@ const MastheadLeftNav = ({
               key={i}
               autoid={autoid}
               selected={rest.selectedMenuItem === link.titleEnglish}
-              navType={rest.navType}>
+              navType={rest.navType}
+              id={`${linkId}-leftnav`}>
```

Comment:
Wondering if we can use an attribute (probably `data-` attribute) rather than `id`, as a precaution to avoid duplicate IDs.

Reply:
@asudoh. I was on the fence about using a `data` attribute, so thanks for suggesting!